Server/src/genericInterfaces/AuthenticationInterfaces.ts

```diff
@@ -21,7 +21,7 @@ export interface IJwtParams {
 export interface IUpdateUserDetail {
     email: string,
     password?: string,
-    organization?: string
+    organizationName?: string
```

Comment:
Good catch here. OrganizationName is how its saved in the DB

Reply:
Now that I think about it.. Its probably better just to have organization than organizationName since its implied in a user table